src/Tribe/RSVP.php

```diff
@@ -219,6 +219,230 @@ public function hooks() {
 		// Has to be run on before_delete_post to be sure the meta is still available (and we don't want it to run again after the post is deleted)
 		// See https://codex.wordpress.org/Plugin_API/Action_Reference/delete_post
 		add_action( 'before_delete_post', array( $this, 'update_stock_from_attendees_page' ) );
+
+		// Handle RSVP AJAX.
+		add_action( 'wp_ajax_nopriv_tribe_tickets_rsvp_handle', [ $this, 'ajax_handle_rsvp' ] );
+		add_action( 'wp_ajax_tribe_tickets_rsvp_handle', [ $this, 'ajax_handle_rsvp' ] );
+	}
+
+	/**
+	 * Handle RSVP processing for the RSVP forms.
+	 *
+	 * @since TBD
+	 */
+	public function ajax_handle_rsvp() {
+		$response = [
+			'html'   => '',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 1 space(s) between "'html'" and double arrow, but found 3.

<sub>hash: 18fc91f8eee943dbac09be10c79b3037</sub>